### PR TITLE
added-learning-resource

### PR DESCRIPTION
**Pull request by Mayank-dev-commits**
Added links to some youtube videos for learning Git in README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,3 +40,8 @@ Once you start feeling more comfortable, you can find more open source projects
 * [Pull Request Roulette](http://www.pullrequestroulette.com/)
 * [Code Triage](https://www.codetriage.com/)
 * [24 P[ULL] R[EQUEST]S](https://24pullrequests.com/)
+
+Links to tutorial on youtube for learning Git:
+* https://www.youtube.com/watch?v=SWYqp7iY_Tc
+* https://www.youtube.com/watch?v=WbwIoQYP6no
+* https://www.youtube.com/watch?v=HVsySz-h9r4
```
Comment (pr4shan7) [inline README.md]: Please do not use bare URLs, instead use the standard markdown format, for example, [name of video, from channel-name ](link)

Comment (pr4shan7) [reply README.md]: Also, these video links can be placed immediately after the resources already present in the README.md.

Comment (pr4shan7) [reply README.md]: I hope you've ensured that these are quality resources and will be helpful for people.

Comment (Mayank-dev-commits) [reply README.md]: Okay I will make changes and these resources are good and also people learn much faster by looking at practical rather than theory

Comment (pr4shan7) [reply README.md]: Yes, you're correct. Videos are much more helpful.

**Comment by pr4shan7** [comment]
@Mayank-dev-commits 
From #15 :

I saw the changes. There are two things:

* You just wrote the names of videos in text, those are no longer links. You can go through the `README.md` to see how I have use markdown syntax to insert links in the document.
* They were placed after `[Your first PR]` while they are expected to be after `[Resources to learn Git]`.

I'm copying this message to #14 also. Please make the required changes and your PR will be merged.

Thanks!

**Comment by pr4shan7** [comment]
If you need any help, you can ask me. @Mayank-dev-commits

**Comment by Mayank-dev-commits** [comment]
Actually I am new to this open source and I am still learning.
I will makes changes as you said .

**Comment by Mayank-dev-commits** [comment]
I have made changes please review it

**Pull request merged**